images-list

```diff
@@ -43,6 +43,7 @@ grafana/grafana rancher/grafana-grafana 7.2.1
 grafana/grafana rancher/mirrored-grafana-grafana 7.1.5
 grafana/grafana-image-renderer rancher/grafana-grafana-image-renderer 2.0.0
 gcr.io/kubebuilder/kube-rbac-proxy rancher/kube-rbac-proxy v0.5.0
+gcr.io/kubebuilder/kube-rbac-proxy rancher/mirrored-kube-rbac-proxy v0.5.0
```

Comment:
Here is link to check when its up: https://hub.docker.com/r/rancher/mirrored-kube-rbac-proxy